### PR TITLE
added post for generating pdf files

### PR DESCRIPTION
**Pull request by ashish-jajoria**

### CODE REVIEW

File: source/_posts/generating-pdf-in-ruby-on-rails.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+title: Generating Pdf in Ruby on Rails
```
Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:2]: title tag character length is less it should be -55 to 60

File: source/_posts/generating-pdf-in-ruby-on-rails.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: use keyword throught the article, which is not seen right now

Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Give link in content and link to previous content or blog

Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Add Video Content or link if Possible

Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: keyword is not linked with the main website, it should be linked with main website where you want to redirect it

Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: blog/2020/02/26/generating-pdf-in-ruby-on-rails/ :  this url is not correct url, it should be in text format with hyphen   Correct URL: http://yourdomain.com/categroy/subcategory/tageted-url

Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: featured imaze size should be 1024 x 600 size atleast, as other blog is having there in nayan blog.

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: What does this mean?

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: If it would have possible then it would have added.

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: I don't know what this means.

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: This can be done by @rana-abhishek

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: I can see some keywords used in the blog

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Done

File: source/_posts/generating-pdf-in-ruby-on-rails.md (hunk 1 above)
Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:2]: Done

File: source/_posts/generating-pdf-in-ruby-on-rails.md (hunk 2 above)
Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Done

Comment (ashish-jajoria) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Done

Comment (Rajeev111974) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Your 1-2 tag should be linked with Nayan website (hyperlink should be there) for redirection to main website

Comment (Rajeev111974) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Good

File: source/_posts/generating-pdf-in-ruby-on-rails.md (hunk 1 above)
Comment (Rajeev111974) [reply source/_posts/generating-pdf-in-ruby-on-rails.md]: fine

File: source/_posts/generating-pdf-in-ruby-on-rails.md (hunk 2 above)
Comment (Rajeev111974) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: it should be placed in there four places if possible, i am finding only in top in the content

File: source/_posts/generating-pdf-in-ruby-on-rails.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
```
Comment (Rajeev111974) [inline source/_posts/generating-pdf-in-ruby-on-rails.md:1]: share button is not redirecting to nayan social page

Comment (Rajeev111974) [reply source/_posts/generating-pdf-in-ruby-on-rails.md:1]: Done

**Pull request merged**